cli/helpers/constants.ts

```diff
@@ -2,3 +2,6 @@ import { PublicKey } from "@solana/web3.js";
 
 export const GUMMYROLL_PROGRAM_ID = new PublicKey("GRoLLMza82AiYN7W9S9KCCtCyyPRAQP2ifBy4v4D5RMD");
 export const GUMMYROLL_CRUD_PROGRAM_ID = new PublicKey("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
+export const BUBBLEGUM_PROGRAM_ID = new PublicKey("BGUMAp9Gq7iTEuizy4pqaxsTyUCBK68MDfK752saRPUY");
+export const GUMBALL_MACHINE_PROGRAM_ID = new PublicKey("GBALLoMcmimUutWvtNdFFGH5oguS7ghUUV6toQPppuTW"); 
+export const CANDY_WRAPPER_PROGRAM_ID = new PublicKey("WRAPYChf58WFCnyjXKJHtrPgzKXgHp6MD9aVDqJBbGh");
```

Comment:
would prefer to import these from `sdk/<program>/` in case we have to change them

```typescript
import { PROGRAM_ID as BUBBLEGUM_PROGRAM_ID } from '../../../contracts/sdk/bubblegum/src/...
```